rust/kcl-lib/e2e/executor/cache.rs

```diff
@@ -259,18 +259,23 @@ extrude(profile001, length = 100)"#
 #[tokio::test(flavor = "multi_thread")]
 async fn kcl_test_cache_add_line_preserves_artifact_commands() {
     let code = r#"sketch001 = startSketchOn(XY)
-  |> startProfile(at = [5.5229, 5.25217])
-  |> line(end = [10.50433, -1.19122])
-  |> line(end = [8.01362, -5.48731])
-  |> line(end = [-1.02877, -6.76825])
-  |> line(end = [-11.53311, 2.81559])
+profile001 = startProfile(sketch001, at = [5.5, 5.25])
+  |> line(end = [10.5, -1.19])
+  |> line(end = [8, -5.5])
+  |> line(end = [-1.02, -6.76])
+  |> line(end = [-11.5, 2.8])
   |> close()
+plane001 = offsetPlane(XY, offset = 20)
```

Comment:
Why did the input kcl need to change?

Reply:
oh

> The sim test output is from fixing SketchOnFace and SketchOnPlane, just something I noticed was broken while investigating this.

never mind

Reply:
The reason I used offsetPlane() was because I wanted to have something to refer to in the second execution, like in way we reproduced #6841.

The coordinate changes were arbitrary. Just wanted rounder numbers to look at.